coordinator/src/routes/health/get-status.ts

```diff
@@ -34,7 +34,8 @@ export default [
     path: "/_status",
     handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
       const checks: { [name: string]: Array<StatusCheckResponse> } = {
-        "validator:status": [await getValidatorHealth()]
+        "validator:status": [await serviceHealthCheck(`${VALIDATOR_HOST}/_status`)],
+        "spine:status": [await spineClient.getStatus()]
```

Comment:
why call this method? would this work:
 serviceHealthCheck(`${spineClient.spineEndpoint}/healthcheck`)